modules/docs/arrow-docs/docs/docs/datatypes/option/README.md

```diff
@@ -138,6 +138,29 @@ val nullableValue: String? = "Hello"
 nullableValue.toOption()
 ```
 
+Some Iterable extensions were added, so you can maintain a friendly API syntax while avoiding null handling (`firstOrNull()`)
+
+```
```

Comment:
We use `ank` for snippets. Check other code blocks in this file to understand them, but tl;dr each snippet block gets executed on builds and the output is then inlined.

The scripting engine we use doesn't quite work for async calls and coroutines, but these would fit just right!